src/slash_assign.js

```diff
@@ -0,0 +1,117 @@
+const core = require("@actions/core");
+const github = require("@actions/github");
+const { Issue, IssueComment } = require("./singleton")
+const { verifyTriageTeam } = require("./helpers")
+
+/**
+ * This function will be only run if
+ * - Event is `issue_comment` and action is `created`
+ * - The messgae is `/assign`
+ * - The issue is not closed
+ *
+ * This function will:
+ * - `/assign @user1` : Can only be used by core-members of the organization. Assigns the issue to user1 
+ * - `/assign` : Assign the issue to the commenter
+ * - prevent new assignees, if the MAX_ASSIGNEE has been reached
+ * 
+ *
+ * @export
+ * @typedef {import('@octokit/core').Octokit & import("@octokit/plugin-rest-endpoint-methods/dist-types/types").Api & { paginate: import("@octokit/plugin-paginate-rest").PaginateInterface; }} octokit
+ * @param {octokit} octokit - Octokit instance
+ * @returns {Promise<void>}
+ */
+
+async function slash_assign( octokit ) {
+
+    const issue = await Issue.getInstance();
+    if (issue.actions_payload.state == "closed") {
+        core.info("Issue is closed, no action needed");
+        return;
+    }
+    if (github.context.eventName === "issue_comment" && github.context.payload.action === "created") {
+        const issue_comment = await IssueComment.getInstance();
+        const issue_comment_body = (issue_comment.details.body ?? "").trim();
+        if((issue_comment_body).trim().startsWith("/assign")) {
```

Comment:
This trim() can be ignored, as trim has been performed at the previous line

```suggestion
        if(issue_comment_body.startsWith("/assign")) {
```